cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -665,6 +665,9 @@ export function changeFrameAsync(
             await job.logger.log(LogType.changeFrame, {
                 from: frame,
                 to: toFrame,
+                // step is passed with sign so we can know the direction
+                step: toFrame - frame,
```

Comment:
We always know the direction because we know ``from``, ``to`` isn't it?

Reply:
Thats true. We can add Math.abs here if we dont need sign of the step. From my perspective having sign helps to undersand direction faster while looking on analytics data.

 What do you think about it?

Reply:
To be honest I just do not see why I would use this information. Usually, when debug, only final frame is important

Reply:
What would be more useful, IMHO, is event duration